real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/GPSPoseMessage.java

```diff
@@ -26,11 +32,12 @@
      * @param longitude of the buggy (negative is West)
      * @param heading   of the buggy (in degrees from North)
      */
-    public GPSPoseMessage(Date timestamp, double latitude, double longitude, double heading) {
+    public GPSPoseMessage(Date timestamp, double latitude, double longitude, double heading, Matrix currentState) {
```

Comment:
Rather than slapping a `Matrix` onto this message, I'd rather we simply integrate the fields into the message